lib/bibliothecary/parsers/maven.rb

```diff
@@ -11,6 +11,8 @@ class Maven
       # "|    \\--- com.google.guava:guava:23.5-jre (*)"
       GRADLE_DEP_REGEX = /(\+---|\\---){1}/
 
+      ANSI_MATCHER = '(\[)?\033(\[)?[;?\d]*[\dA-Za-z]([\];])?'
```

Comment:
This should match the ANSI codes.

Reply:
I wonder if we should use a gem here? https://github.com/fazibear/colorize ?

Reply:
I went with https://github.com/piotrmurach/strings-ansi to handle this instead. I had originally pulled the regex from this project and it seems to work where the `colorize` gem wasn't picking up all the codes.